package.json

```diff
@@ -33,8 +33,8 @@
     "@babel/eslint-parser": "^7.18.2",
     "@kubernetes/client-node": "^0.16.2",
     "@mojaloop/api-snippets": "^17.0.3",
-    "@mojaloop/sdk-standard-components": "^17.0.3",
-    "@pm4ml/mcm-client": "^2.0.11",
+    "@mojaloop/sdk-standard-components": "^17.1.3",
+    "@pm4ml/mcm-client": "github:pm4ml/mcm-client#feat/IPROD-93",
```

Comment:
Will be updated to `"@pm4ml/mcm-client": "^3.0.0",` after merging **mcm-client** to _master_ branch